Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -319,6 +335,8 @@ static void HandleQuery()
     return SendReply("l");
   else if (!strncmp((const char*)(s_cmd_bfr), "qThreadExtraInfo", strlen("qThreadExtraInfo")))
     return SendReply("00");
+  else if (!strncmp((const char*)(s_cmd_bfr), "qHostInfo", strlen("qHostInfo")))
```

Comment:
I don't like this chunk of C code (`(const char*)(s_cmd_bfr)` should be assigned to a local std::string_view variable to avoid repeating the ugly casts, strncmp and string literals), but I guess we can clean this up in a follow-up PR.

Reply:
So in #10319 , we are moving these to a helper method which still has the stenxmp. Because I did it in that pr, it might not be a great idea to do it here.

Reply:
Yeah, I'm not saying we should change this in this PR